src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/Signing/TrustedSignersCommand.cs

```diff
@@ -23,107 +22,305 @@ internal static class TrustedSignersCommand
+
+                    syncCommand.OnExecute(async () =>
+                    {
+                        return await ExecuteCommand(TrustCommand.Sync, algorithm: null, allowUntrustedRootOption: false, owners: null, verbosity, configFile, getLogger, setLogLevel, name: name.Value);
+                    });
+                });
+
+                trustedSignersCmd.Command("remove", (syncCommand) =>
+                {
+                    syncCommand.Description = Strings.TrustRemoveCommandDescription;
+                    CommandOption configFile = syncCommand.Option(
+                        "--configfile",
+                        Strings.Option_ConfigFile,
+                        CommandOptionType.SingleValue);
+
+                    syncCommand.HelpOption(XPlatUtility.HelpOption);
+
+                    CommandOption verbosity = syncCommand.VerbosityOption();
+
+                    CommandArgument name = syncCommand.Argument("<NAME>",
+                                               Strings.TrustedSignerNameToRemove);
+
+                    syncCommand.OnExecute(async () =>
+                    {
+                        return await ExecuteCommand(TrustCommand.Remove, algorithm: null, allowUntrustedRootOption: false, owners: null, verbosity, configFile, getLogger, setLogLevel, name: name.Value);
+                    });
+                });
+
+                trustedSignersCmd.Command("author", (authorCommand) =>
+                {
+                    authorCommand.Description = Strings.TrustAuthorCommandDescription;
+
+                    CommandOption allowUntrustedRootOption = authorCommand.Option(
+                                                "--allow-untrusted-root",
+                                                Strings.TrustCommandAllowUntrustedRoot,
+                                                CommandOptionType.NoValue);
+
+                    CommandOption configFile = authorCommand.Option(
+                        "--configfile",
+                        Strings.Option_ConfigFile,
+                        CommandOptionType.SingleValue);
+
+                    authorCommand.HelpOption(XPlatUtility.HelpOption);
 
-                CommandOption configFile = trustedSignersCmd.Option(
+                    CommandOption verbosity = authorCommand.VerbosityOption();
+
+                    CommandArgument name = authorCommand.Argument("<NAME>",
+                                               Strings.TrustedSignerNameToAdd);
+                    CommandArgument package = authorCommand.Argument("<PACKAGE>",
+                                               Strings.TrustLocalSignedNupkgPath);
+
+                    authorCommand.OnExecute(async () =>
+                    {
+                        return await ExecuteCommand(TrustCommand.Author, algorithm: null, allowUntrustedRootOption.HasValue(), owners: null, verbosity, configFile, getLogger, setLogLevel, name: name.Value, sourceUrl: null, packagePath: package.Value);
```

Comment:
If we change `packagePath` to accept multiple values then we have to pass `packagePath.Values`.

Reply:
See https://github.com/NuGet/Home/issues/10647#issuecomment-801523491